api/python/provisioner/inputs.py

```diff
@@ -467,54 +467,61 @@ class Release(ParamGroupInputBase):
 
 
 class StorageEnclosureParams():
-    _param_group = 'storage'
-    enclosure_id: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr="Enclosure ID"
-    )
+    _param_group = 'storage_enclosure'
     type: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Type of storage"
+        _param_group, descr="Type of storage"
     )
-    primary_ip: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller A IP",
+    controller_primary_mc_ip: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Controller A IP",
         validator=Validation.check_ip4
     )
-    secondary_ip: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller B IP",
+    controller_secondary_mc_ip: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Controller B IP",
         validator=Validation.check_ip4
     )
     controller_user: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller user"
+        _param_group, descr="Controller user"
     )
     # TODO IMPROVE EOS-14361 mask secret
     controller_secret: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller password"
+        _param_group, descr="Controller password"
     )
     controller_type: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller type"
+        _param_group, descr="Controller type"
+    )
+    storage_metadata_device: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Storage Metadata Device"
+    )
+    storage_data_devices: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Storage Data Devices"
+    )
+    hostname: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Node Hostname"
     )
 
 
 @attr.s(auto_attribs=True)
 class StorageEnclosure(ParamGroupInputBase):
-    controller_a_ip: str = StorageEnclosureParams.primary_ip
-    controller_b_ip: str = StorageEnclosureParams.secondary_ip
+    type: str = StorageEnclosureParams.type
+    controller_a_ip: str = StorageEnclosureParams.controller_primary_mc_ip
+    controller_b_ip: str = StorageEnclosureParams.controller_secondary_mc_ip
     controller_user: str = StorageEnclosureParams.controller_user
     controller_secret: str = StorageEnclosureParams.controller_secret
+    storage_metadata_device: str = StorageEnclosureParams.storage_metadata_device
+    storage_data_devices: str = StorageEnclosureParams.storage_data_devices
+    hostname: str = StorageEnclosureParams.hostname
 
 
 class NodeNetworkParams():
     _param_group = 'node'
-    hostname: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr="node hostname"
-    )
-    roles: List = ParamGroupInputBase._attr_ib(
-        _param_group, descr="List of roles assigned to the node"
+    type: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Common Server configuration"
```

Comment:
shouldn't we use a word `type` in the description in some way, for now description doesn't provide any understanding of the parameter

Reply:
I meant that `Common Server configuration` looks too generic. I believe it should be something like `type of smth`

Reply:
Yes, removed this param from nodes altogether. It was not necessary here.